principles/coding-principles.md

```diff
@@ -15,8 +16,10 @@ They are guidance, not The Law - there will always be edge cases, but you should
 
 Correct means provably correct - with tests. All fixes & new features should include tests to prevent regressions.
 Choose clarity over cleverness.
-Don’t Repeat Yourself - The ‘[Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))’ is a good approach to managing duplication. Less code is usually better, but not at the expense of clarity - Clean Coding techniques can be used to ensure the code remains correct, clear and concise.
-Leave things better than you found them - Boyscout the code
+* Don’t Repeat Yourself - The ‘[Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))’ is a good approach to managing duplication. 
+* Less code is usually better, but not at the expense of clarity - Clean Coding techniques can be used to ensure the code remains correct, clear and concise.
+* Leave things better than you found them - Boyscout the code
+* Follow [SOLID principles](https://en.wikipedia.org/wiki/SOLID). This will make code more understandable and easier to maintain
 
```

Comment:
I'm a big pusher of no comments in code - as mentioned above the code should be clear and concise and also self describing. Comments that are added are never then updated to reflect changes made in further pieces of work.

Reply:
I agree, edge cases exist but in the main the code should be self-describing and refactored if it isn't, as per Clean Coding techniques